state/cassandra/metadata.yaml

```diff
@@ -0,0 +1,76 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: state
+name: cassandra
+version: v1
+status: stable
+title: "Apache Cassandra"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-cassandra/
+capabilities:
+  # If actorStateStore is present, the metadata key actorStateStore can be used
+  - crud
+  # Docs mention TTL but the code neither advertises these nor shows any support for it.
```

Comment:
Oops!  But the code handles it indeed!

https://github.com/dapr/components-contrib/blob/a6eb406fc20530fc784b44dddee3d36da9eac69b/state/cassandra/cassandra.go#L282-L289